scripts/basic-test.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 $PYTEST --runpytest=subprocess tests/pytest
 
-pip install .[datetime]
+pip install ".[datetime]"
```

Comment:
Not for now, but this reminds me that I think we should probably get rid of our use of extras in tests and start using a requirements.txt for them.